src/main/java/mil/dds/anet/config/AnetConfiguration.java

```diff
@@ -146,6 +183,56 @@ public void setDictionary(Map<String, Object> dictionary) {
     this.dictionary = Collections.unmodifiableMap(dictionary);
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
Please put markers like @SuppressWarnings("unchecked") as much as possible on the relevant assignment instead of on the whole method (which may lead to other suppressions that you then don't notice).